File: api/code-samples/python/klaytnDebug/profiling/startPProf.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

address = "localhost"
port = 6060

sdk = OpenSDK(host)
debug_response = sdk.debug.start_p_prof(address, port)

print(json.loads(debug_response.response.data))

File: web3rpc/rpc-specs/api/code-samples/javascript/debug/profilling/startPProf.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const address = "localhost";
    const port = 6060;
    
    sdk.debug.startPProf({address, port}, (err, data, response) => {
        console.log(data);
    });
}
)()